client/state/selectors/get-is-woo-passwordless.ts

```diff
@@ -14,6 +14,11 @@ export default function getIsWooPasswordless( state: AppState ): boolean {
 		return false;
 	}
 
+	// Enable Woo Passwordless if user is from WooCommerce Core Profiler.
+	if ( state?.route?.query?.initial?.from === 'woocommerce-core-profiler' ) {
```

Comment:
We can reuse `isWooCommerceCoreProfilerFlow` selector here.

https://github.com/Automattic/wp-calypso/blob/eef6730df07f7816be791932cfbab18184731021/client/state/selectors/is-woocommerce-core-profiler-flow.ts#L12-L17

Reply:
Thank you! Updated in [use isWooCommerceCoreProfilerFlow](https://github.com/Automattic/wp-calypso/pull/93230/commits/3a09af66d597209a263a704430f80fb325522c4e)